server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -86,15 +86,16 @@ export class DeliLambdaFactory
 		};
 
 		let gitManager: IGitManager;
-		let document: IDocument;
+		let document: IDocument | undefined;
 
 		try {
 			// Lookup the last sequence number stored
 			// TODO - is this storage specific to the orderer in place? Or can I generalize the output context?
-			document = await this.documentRepository.readOne({ documentId, tenantId });
+			document =
+				(await this.documentRepository.readOne({ documentId, tenantId })) ?? undefined;
```

Comment:
Is the null coallesing operator here because readOne could return `null`? Might be worth calling out unless it's a super common pattern in server. `?? undefined` looks really weird to my eyes.

Reply:
I was probably a bit overzealous here and should have reduced scope by keeping it null rather than using `?? undefined`. I was trying to avoid the ESLint warning for `@rushstack/no-new-null`, but there were so many of these cases I gave up.

I still think it is better to explicitly convert `null`, which is returned by various storage dependencies like MongoDB and Redis, to `undefined`, but it ended up being a larger task than I wanted to handle.

Reply:
Yeah it might be OK to disable that rule for the most part while you're in "cleanup mode".